web/modules/custom/hoeringsportal_citizen_proposal/src/Form/ProposalFormBase.php

```diff
@@ -129,6 +129,7 @@ protected function getDefaultFormValues(): array {
     return [
       'name' => $entity?->field_author_name->value ?? $userData['name'] ?? NULL,
       'email' => $entity?->field_author_email->value ?? $userData['email'] ?? NULL,
+      'email_display' => $entity?->field_author_email_display->value ?? $userData['email_display'] ?? NULL,
```

Comment:
`$userData` holds data from MitID authentication and we don't get an `email_display` value from that.

Reply:
Changed